components/core/src/clp_s/search/Output.cpp

```diff
@@ -137,23 +137,23 @@ void Output::init(
 
     for (auto column_reader : column_readers) {
         auto column_id = column_reader->get_id();
-        if ((0 != m_wildcard_type_mask
-             && 0
-                        != (m_wildcard_type_mask
-                            & node_to_literal_type(m_schema_tree->get_node(column_id).get_type())))
+        if ((0
+             != (m_wildcard_type_mask
+                 & node_to_literal_type(m_schema_tree->get_node(column_id).get_type())))
             || m_match.schema_searches_against_column(schema_id, column_id))
         {
             ClpStringColumnReader* clp_reader = dynamic_cast<ClpStringColumnReader*>(column_reader);
             VariableStringColumnReader* var_reader
                     = dynamic_cast<VariableStringColumnReader*>(column_reader);
+            DateStringColumnReader* date_reader
+                    = dynamic_cast<DateStringColumnReader*>(column_reader);
             if (clp_reader != nullptr && clp_reader->get_type() == NodeType::ClpString) {
                 m_clp_string_readers[column_id].push_back(clp_reader);
             } else if (var_reader != nullptr && var_reader->get_type() == NodeType::VarString) {
```

Comment:
Do you want to change `clp_reader != nullptr` and `var_reader != nullptr` to `nullptr != clpreader` and `nullptr != var_reader`?

Reply:
Sure, might as well